Code/Python/trapping_rainwater.py

```diff
@@ -0,0 +1,64 @@
+# Problem Description:

```

Comment:
use ''' and the comments into multiline comments instead of using # again and again